### PR TITLE
IE-356 + Added shorthand for specifying a 'since' based on duration

### PR DESCRIPTION
**Pull request by dspasojevic**
Add a new argument to the `logs` command:

```
      --within         The start date of the logs to retrieve, expressed as a
                       duration to be applied to now (e.g. 1d/P1D for 1 day,
                       2w/P2W for two weeks).
```

that makes it easier to retrieve some of the most recent log records.

### CODE REVIEW

**Review by desimpson — APPROVED** [review]
LGTM.

File: src/scripts/logs.ts [hunk 1]
```diff
@@ -103,14 +115,27 @@ export default (
...
       })
       .option('until', {
-        type: 'string',
         description: 'The end date of the logs to retrieve.',
-        coerce: (date: string) => readonlyDate(date),
+        coerce: (date: string) => coercedDate(date),
+      })
+      .option('within', {
+        description:
+          'The start date of the logs to retrieve, expressed as a duration to be applied to now (e.g. 1d/P1D for 1 day, 2w/P2W for two weeks).',
```
Comment (desimpson) [inline src/scripts/logs.ts:127]: **suggestion**: Perhaps we could add a link here to e.g., https://en.wikipedia.org/wiki/ISO_8601#Durations, to benefit people unfamiliar with ISO 8601 duration formats?

**Review by desimpson — CHANGES_REQUESTED** [review]
Left one suggestion. Otherwise LGTM.

File: src/scripts/logs.ts (hunk 1 above)
Comment (dspasojevic) [reply src/scripts/logs.ts]: Done.

**Comment by dspasojevic** [comment]
Ready for re-review @desimpson .

**Review by desimpson — APPROVED** [review]

**Pull request merged**